### ISSUE TITLE
Low accuracy on UCF Sports Action dataset.

### ISSUE THREAD

**Issue by ArchieGu** [kenshohara/3D-ResNets-PyTorch#85]
Hi,
I am using the UCF Sports Action dataset which is kind of a subset of UCF101.
And I followed the instruction in Readme file to prepare the dataset.
However after I trained 200 epoch, the accuracy of both train and test are pretty low (30%train and 22% test)

Could you give me some advices about the data preparing or the possible reason about the low accuracy?

**Comment by ArchieGu** [comment]
No, I didn't use the finetuning models yet. I will try it asap.

By the way, what is the accuracy you achieved on UCF101 dataset by using none finetuning model on resnet 34?

I can only achieve accuracy around 38% on validation, and the training accuracy can achieve around 80%. I think it goes overfitting. Do you have any suggestion about this?

Thank you .

**Comment by kenshohara** [comment]
When I train ResNet-18 on UCF-101 from scratch,
I got around 80% and 40% accuracies on the training and validation sets, respectively.
Because UCF-101 is a relatively small dataset to train 3D CNNs from scratch,
it causes overfitting.
I think the training on UCF-Sports is similar.

**Comment by ArchieGu** [comment]
Oh, right, UCF-Sports is really a very small dataset. I think I can train the model using UCF-101 and using UCF-Sports to test the model.

**Comment by ArchieGu** [comment]
One more question, why I cannot using resnet 101 to train the model? Each time I set depth !=34, the program gives me an error about illegal access to memory.

**Comment by ArchieGu** [comment]
@kenshohara 

BTW, the UCF sports action dataset has only 10 classes , but the pre trained model has 400 classes.
How can I use the pre trained model to train my data??  They are even not the same dataset.
